Lib/_pyrepl/fancycompleter.py

```diff
@@ -131,6 +130,7 @@ def attr_matches(self, text):
         if prefix and prefix != attr:
             return [f'{expr}.{prefix}']  # autocomplete prefix
 
+        names.sort()
```

Comment:
This does not sort the `values`. Maybe revert the commit that introduced this if it is too much trouble to fix this.

Reply:
yes, pro tip for my future self: don't write code when you are sleepy :).
commit 7d2c7902817 reverts it, and also adds a comment to explain WHY we need to sort words.